src/Omega_h_map.hpp

```diff
@@ -22,6 +22,16 @@ void map_into_range(
 template <typename T>
 Read<T> map_onto(Read<T> a_data, LOs a2b, LO nb, T init_val, Int width);
 
+/** 
+ * \brief return the array of b_data in the order specified by a2b
+ * \remark a2b does not need to use all entries of b_data; it
+ *         can replicate values from b_data by using an index
+ *         multiple times.
+ * \param a2b (in) specifies output order of data from b_data
+ * \param b_data (in) data to reorder, size = size(a2b)*width
+ * \param width (in) number of entries in b_data per item
+ * \return reordered array of b_data
```

Comment:
looks good, The \brief 'reorder and associate b_data with a_data as per the order specified in a2b'
\return 'a_data (i.e. the reordered array of b_data)'

Reply:
This is for `unmap`.  Were you looking at `map_onto`?

Reply:
unmap:
https://github.com/SCOREC/omega_h/blob/00131eb0133796f39ff2ecd0962d9b3d14d5887b/src/Omega_h_map.cpp#L81-L88

Reply:
OK.  Thanks for the feedback.  I prefer to only reference the named input arguments to avoid confusion.